sdcm/keystore.py

```diff
@@ -98,6 +98,9 @@ def get_azure_credentials(self):
     def get_argusdb_credentials(self):
         return self.get_json("argusdb_config_v2.json")
 
+    def get_argus_rest_credentials(self):
+        return self.get_json("argus_rest_api.json")
```

Comment:
maybe the file name should be `argus_rest_credentials.json` as the function name ? to avoid confusing ?

Reply:
Done